frontend/src/pages/nft/nft.module.scss

```
@@ -107,4 +107,35 @@
     overflow: hidden;
     text-overflow: ellipsis;
   }
+
+  .metadata {
+    padding: 12.5px 0 11.5px 0;
+    &Item {
+      svg {
+        min-width: 24px;
+        height: 24px;
+      }
+      &:not(:last-child)::after {
+        display: block;
+        content: '';
+        width: 296px;
```

```suggestion
        width: 50%;
```

Let's make it responsive.